gulp/plugins/gulp-react-docgen.js

```diff
@@ -52,9 +52,13 @@ export default (filename) => {
           type: name,
         }
       })
+
+      parsed.path = file.path
+        .replace(`${process.cwd()}/`, '')
+        .replace(new RegExp(_.escapeRegExp(path.sep), 'g'), '/')
       parsed.props = _.sortBy(parsed.props, 'name')
 
-      result[relativePath] = parsed
+      result[componentName] = parsed
```

Comment:
`posixPath` is now calculates with plugin. For `key` in `docgenInfo.json` will be used a component name, it's unique, so we don't to regexps to find the component info :v:

Reply:
We need to make it windows compatible, so it needs to use Noed's `path` module for path modification.  We cannot hardcode forward or backward slashes.

Reply:
I've shipped a small change, looks correct. I don't have a windows PC near and I'm not familar with `path`. If you have better solution, feel free to push updates.